ros_buildfarm/templates/doc/rosdoc2_job.xml.em

```diff
@@ -167,6 +167,8 @@ else:
         'sleep 1',
         '',
         'echo "# BEGIN SECTION: Build Dockerfile - doc"',
+	'# copy rosdoc2 into image build context directory',
+	'cp -r $WORKSPACE/rosdoc2 $WORKSPACE/docker_doc/rosdoc2',
```

Comment:
In order to be available in the docker context to be copied into the image the `rosdoc2` source directory needs to be in the docker context subdirectory.